packages/botonic-nlp/src/parser/data-augmenter.ts

```diff
@@ -0,0 +1,60 @@
+import { Augmenter } from './types'
+
+export class DataAugmenter {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  static KEYWORD_PATTERN = /\[([^\[\]\(\)]*?)\][^\(]/
```

Comment:
add const
Please investigate if we can tweak eslint rule https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md so that it accepts const static fields with capitals